src/styles/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-size: center;
```

Comment:
The 'background-size' property should have a size value (like 'contain', 'cover', etc.) or explicit width and height, 'center' is not a valid value for 'background-size'. It's possible you intended to use 'background-position: center;' which is already defined in line 11.